test/cql-pytest/test_materialized_view.py

```diff
@@ -740,3 +740,25 @@ def test_mv_override_clustering_order_error(cql, test_keyspace):
         with pytest.raises(InvalidRequest, match="CLUSTERING ORDER BY"):
             with new_materialized_view(cql, table, '*', 'p, c, x', 'p is not null and c is not null and x is not null', 'with clustering order by (c ASC, x ASC, z DESC)') as mv:
                 pass
+
+# Test views that only refer to the primary key, exercising the invisible
+# empty type columns that are injected into the view schema in order to
+# compute the view row liveness.
+def test_mv_with_only_primary_key_rows(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'id int PRIMARY KEY, v1 int, v2 int') as base:
+        # Use a synchronous view so we don't have to worry about races between flush and
+        # view updates.
+        with new_materialized_view(cql, table=base, select='id', pk='id', where='id IS NOT NULL',
+                extra='WITH synchronous_updates = true') as view:
```

Comment:
This test fails on Cassandra, because "synchronous_updates" is a Scylla-only invention, and even without it, Cassandra doesn't have the "virtual columns" liveness fix that we have so will likely fail on this test (I didn't check). So I think you should add the `scylla_only` fixture to this test.